selectable/templatetags/selectable_tags.py

```diff
@@ -13,5 +13,5 @@ def include_jquery_libs(version='1.11.2', ui='1.11.3'):
 
 
 @register.inclusion_tag('selectable/jquery-css.html')
-def include_ui_theme(theme='base', version='1.11.3'):
+def include_ui_theme(theme='smoothness', version='1.11.3'):
```

Comment:
The jquery-ui on the version on `pipy` is `1.8.23` and it comes with the `base` theme, but the selectable version from the project uses jquery-ui version `1.11.3`, which `base` theme doesn't available.